modules/vpn_ha/main.tf

```diff
@@ -106,6 +106,8 @@ resource "google_compute_router_peer" "bgp_peer" {
   router          = local.router
   peer_ip_address = each.value.bgp_peer.address
   peer_asn        = each.value.bgp_peer.asn
+  enable          = lookup(each.value, "admin_enabled", true)
```

Comment:
Looks like we are iterating on `var.tunnels` which is a map(object()). Wouldn't we need to add `admin_enabled` as an attrib in that object? Otherwise this would always evaluate to true.